docs/CHANGELOG.md

```diff
@@ -10,6 +10,22 @@ nav_order: 5
 
 ## main
 
+* BREAKING: Remove deprecated support for loading ViewComponent engine manually.
```

Comment:
Maybe we can add an explanation directly in the changelog? Something like:

```suggestion
* BREAKING: Remove deprecated support for loading ViewComponent engine manually. Make sure you removed `require "view_component/engine"` from your `Gemfile`.
```

Rationale: it's nice to be able to fix that kind of thing by reading the changelog entry for the major releases.